Bottom: c65826557a47314de9efdbf436128ec22315a308
Top:    0689a5a094e8abbda34844d11ad9dc11d7ffc532
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-28 22:14:07 -0400

Refresh of temp1

---

diff --git a/sound/soc/fsl/mpc5200_psc_ac97.c b/sound/soc/fsl/mpc5200_psc_ac97.c
index cc383a5..e417582 100644
--- a/sound/soc/fsl/mpc5200_psc_ac97.c
+++ b/sound/soc/fsl/mpc5200_psc_ac97.c
@@ -49,7 +49,6 @@ static unsigned short psc_ac97_read(struct snd_ac97 *ac97, unsigned short reg)
 	unsigned int val;
 
 	spin_lock(&psc_dma->lock);
-	//printk("ac97 read: reg %04x\n", reg);
 
 	/* Wait for it to be ready */
 	timeout = 1000;
@@ -84,8 +83,6 @@ static unsigned short psc_ac97_read(struct snd_ac97 *ac97, unsigned short reg)
 	}
 	val = (val >> 8) & 0xffff;
 
-	//printk("ac97 read ok: reg %04x  val %04x\n", reg, val);
-
 	spin_unlock(&psc_dma->lock);
 	return (unsigned short) val;
 }
@@ -95,7 +92,6 @@ static void psc_ac97_write(struct snd_ac97 *ac97, unsigned short reg, unsigned s
 	struct psc_dma *psc_dma = ac97->private_data;
 	int timeout;
 
-	//printk("ac97 write: reg %04x  val %04x\n", reg, val);
 	spin_lock(&psc_dma->lock);
 
 	/* Wait for it to be ready */
@@ -119,8 +115,6 @@ static void psc_ac97_cold_reset(struct snd_ac97 *ac97)
 {
 	struct psc_dma *psc_dma = ac97->private_data;
 
-	printk("psc_ac97_cold_reset %p\n", ac97);
-
 	/* Do a cold reset */
 	out_8(&psc_dma->psc_regs->op1, MPC52xx_PSC_OP_RES);
 	udelay(10);
@@ -365,7 +359,6 @@ static int __devinit psc_ac97_of_probe(struct of_device *op,
 		iounmap(regs);
 		return -ENOMEM;
 	}
-	printk("psc_dma %p\n", psc_dma);
 
 	spin_lock_init(&psc_dma->lock);
 	psc_dma->irq = irq;
